src/docs/content/observability/observability-scenario/pre-requisites/setup.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 1
+tags:
+  - mushop setup
+---
+
+## Pre-requisites
+
+### Install Ingress Controller
+
+```bash
+kubectl create namespace mushop
+```
+```text
+
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/mandatory.yaml
```

Comment:
We should not include specific version. This including is an old version that is not compatible with all features from other services.

Reply:
took it from here: https://docs.cloud.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengsettingupingresscontroller.htm